resources/extract-tagged.py

```diff
@@ -6,6 +6,7 @@ def extract_tagged(target, tag):
     import re
     start = re.compile('^(\s+).+tag::' + tag)
     end = re.compile('end::' + tag)
+    callout = re.compile(r'// (<[^>]+>)')
```

Comment:
to make this safer, what about:

    callout = re.compile(r'// (<[^>]+>)\s*?\n')